enhancements/machine-api/spot-instances.md

```diff
@@ -75,6 +75,10 @@ the following requirements for integration will work for each of AWS, Azure and
 - If the Spot request cannot be satisfied when a Machine is created, the Machine will be marked as failed.
   This Machine would be remediated by an MHC if present.
 
+- The actuator should label Machines/Nodes based on their lifecycle
```

Comment:
IMO, we should not add labels automatically that affect scheduling.  For one, in the future, there might be some upstream label for this kind of thing (if there isn't already one now, IDK).  For two, users can add this label if they want to distinguish their spot workers from others.  We should do as little as possible labeling and let users define what they think is appropriate.